imqsauth/http.go

```diff
@@ -614,11 +630,67 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 		} else {
 			central.Central.Log.Errorf("Could not set group roles for %v: %v", groupname, err)
 			authaus.HttpSendTxt(w, http.StatusNotAcceptable, fmt.Sprintf("Could not set group roles for %v: %v", groupname, err))
+			return
 		}
 	} else {
 		central.Central.Log.Warnf("Group '%v' not found: %v", groupname, e)
 		authaus.HttpSendTxt(w, http.StatusNotFound, fmt.Sprintf("Group '%v' not found: %v", groupname, e))
+		return
 	}
+
+	broadCastGroupChange(central, r, groupname)
+}
+
+func broadCastGroupChange(central *ImqsCentral, r *httpRequest, groupname string) {
```

Comment:
nitpicky: `broadcast` is one word, so not `broadCast`.
Also, avoid empty line at start of function.